ui/app/components/app/modals/customize-nonce/customize-nonce.component.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Modal from '../../modal';
+import TextField from '../../../ui/text-field';
+import Button from '../../../ui/button';
+import Typography from '../../../ui/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+  ALIGN_ITEMS,
+  BLOCK_SIZES,
+} from '../../../../helpers/constants/design-system';
+import Box from '../../../ui/box';
+import withModalProps from '../../../../helpers/higher-order-components/with-modal-props';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const CustomizeNonce = ({
+  hideModal,
+  customNonceValue,
+  nextNonce,
+  updateCustomNonce,
+  getNextNonce,
+}) => {
+  const [customNonce, setCustomNonce] = useState('');
+  const t = useI18nContext();
+
+  const renderModalContent = (
+    <div className="customize-nonce-modal">
+      <div className="customize-nonce-modal__main-header">
+        <Typography variant={TYPOGRAPHY.H4} fontWeight={FONT_WEIGHT.BOLD}>
+          {t('editNonceField')}
+        </Typography>
+        <button
+          className="fas fa-times customize-nonce-modal__close"
+          title={t('close')}
+          onClick={hideModal}
+        />
+      </div>
+      <div className="customize-nonce-modal__description">
```

Comment:
This div can be replaced with `<Box marginTop={2}>` and the corresponding css class can be removed